user_data.tpl

```diff
@@ -0,0 +1,9 @@
+#cloud-config
+users:
+  - default
+  - name: debian
+    sudo: ["ALL=(ALL) NOPASSWD:ALL"]
+    ssh_pwauth: True
```

Comment:
Why do we need to enable pwd auth for this user?  I thought we needed to avoid that for security reasons.